Objects/abstract.c

```diff
@@ -2839,8 +2837,8 @@ PyObject_GetAIter(PyObject *o) {
     PyObject *it = (*f)(o);
     if (it != NULL && !PyAIter_Check(it)) {
         PyErr_Format(PyExc_TypeError,
-                     "aiter() returned not an async iterator of type '%.100s'",
```

Comment:
This existing error is pretty confusing, I'd reword it as `__aiter__ must return async iterator (got %T)`.